library/extractor/src/main/java/com/google/android/exoplayer2/extractor/mp4/AtomParsers.java

```diff
@@ -1173,6 +1173,17 @@ private static void parseVideoSampleEntry(
       } else if (childAtomType == Atom.TYPE_vpcC) {
         ExtractorUtil.checkContainerInput(mimeType == null, /* message= */ null);
         mimeType = (atomType == Atom.TYPE_vp08) ? MimeTypes.VIDEO_VP8 : MimeTypes.VIDEO_VP9;
+        parent.setPosition(childStartPosition + Atom.HEADER_SIZE);
+        // vpcC atom parsing based on FFmpeg implementation
+        // see https://github.com/FFmpeg/FFmpeg/blob/master/libavformat/vpcc.c
+        parent.skipBytes(6);
```

Comment:
I think we should add comment saying which fields are skipped here.

Reply:
Thanks for the suggestion! Please take another look.